internal/proxy/task_scheduler.go

```diff
@@ -480,7 +480,7 @@ func (sched *taskScheduler) definitionLoop() {
 		case <-sched.ddQueue.utChan():
 			if !sched.ddQueue.utEmpty() {
 				t := sched.scheduleDdTask()
-				sched.processTask(t, sched.ddQueue)
+				go sched.processTask(t, sched.ddQueue)
```

Comment:
Should use a routine pool be used to control concurrency?

Reply:
this pr is a quick fix, which will cherry-pick to 2.2 branch. and a new task scheduler which remove the task queue and control the concurrency will be a large change, which will be done in next pr for 2.3 branch only